### PR TITLE
Update protobuf, grpcio and clap

### LINKED ISSUE #160: Update protobuf to 2.0
The hardest part here might be the internal third_party update to Lyft.

### PR DESCRIPTION
**Pull request by SirVer**
Updates

- clap to get rid of a build warning.
- protobuf to 2.0 and grpcio to 0.4. This allowed to to simplify build steps slightly by not needing to shell out to `protoc` ourselve, but use `rustc-grpcio` to do the heavy lifting for us. 

This also fixes CI.

Subsumes #159 by including the change and fixes #160. 

PAIR=@kerrick-lyft

### CODE REVIEW

File: point_viewer_grpc_proto_rust/build.rs [hunk 1]
```diff
@@ -61,33 +42,40 @@ fn wrap_in_module(contents: String, mod_name: &str) -> String {
...
+        &["point_viewer_grpc_proto_rust/src/proto.proto"],
+        &[git_repo_root.clone()],
+        &out_dir
+    ).expect("Failed to compile gRPC definitions!");
 
-    let mut child = cmd.spawn().unwrap();
-    if !child.wait().unwrap().success() {
-        panic!("protoc exited with non-zero exit code");
-    }
+    // NOCOM(#sirver): what
```
Comment (thomasschiwietz) [inline point_viewer_grpc_proto_rust/build.rs]: nit: remove commented lines

**Comment by SirVer** [comment]
All done. Thank you!

@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Comment by kerrick-lyft** [comment]
Thanks for doing this! I will take care of updating the rest of Lyft's codebase to use the new GRPC version. I think that getting our Rust packages on the latest GRPC is actually the hard part; the rest should hopefully just involve a version bump.

**Pull request merged**